src/generic/numberparsing.h

```diff
@@ -507,7 +1450,6 @@ really_inline bool parse_number(UNUSED const uint8_t *const src,
     exponent += (neg_exp ? -exp_number : exp_number);
   }
   if (is_float) {
-    uint64_t power_index = 308 + exponent;
     if (unlikely((digit_count >= 19))) { // this is uncommon
```

Comment:
just fyi if we ever need it in the future, in this case we can use a modified form of `compute_float_64` and get almost as fast performance as for smaller numbers get with that function. but obviously that would require oddly long numbers for json

Reply:
I'll add a note.